feat: move credential docs into concept section (#85)
KILTprotocol/docs: docs/concepts/credentials/0_overview.md

---
id: overview
title: Credentials - Overview
---

**Credentials** are attested claims which are owned by a **Claimer**, attested by an **Attester** and can be verified by a **Verifier**.

To get a Credential a Claimer needs to go through following process:
1. Find a **CType**, a Claim should be based on (Can be advertised by the potential Attester and Verifier)
2. Make a **Claim**
3. Potentially request and receive **Terms** and agree on a **Quote** with an Attester
4. **Request an Attestation**

> 5. Wait for the Claim to be **attested** on-chain by the Attester

Reviewer: Would not put too many details in here.

```suggestion
5. Wait for the Claim to be **attested** by the Attester
```

Author: de5620fe

To verify the Credential a Claimer can send a presentation of it to a Verifier

> 1. The Verifier may request a **Credential** as the first step, providing a nonce for security reasons

Reviewer: Same here, probably does not add anything valuable considering this is an introductory section, does it?

```suggestion
1. The Verifier may request a **Credential** as the first step
```

Author: de5620fe

> 2. The Claimer selects properties he wants to **disclose** and signs the Presentation of the Credential including the nonce

Reviewer: ```suggestion
2. The Claimer selects properties they want to **disclose** and signs the Presentation of the Credential
```

Author: de5620fe

3. The Claimer sends the **Presentation** to the Verifier

> 4. The Verifier **checks** the signature, nonce, claim contents and decides, if he trusts the Attester

Reviewer: ```suggestion
4. The Verifier **checks** its structure and signature and decides, if they trust the Attester
```

Author: de5620fe
